sci-mathematics/octave/octave-6.4.0-r1.ebuild

```diff
@@ -29,6 +29,7 @@ RDEPEND="
 	app-text/ghostscript-gpl
 	sys-apps/texinfo
 	dev-libs/libpcre:3=
+	media-gfx/graphicsmagick
```

Comment:
This is unlikely a hard-dependency on graphicsmagick. See also https://bugs.gentoo.org/314431

Reply:
Thank you for your suggestion. I have force pushed a new ebuild.